nav2_dynamic_params/src/example_dynamic_params.cpp

```diff
@@ -64,22 +65,53 @@ int main(int argc, char ** argv)
   param_validator->add_param("foo", rclcpp::ParameterType::PARAMETER_DOUBLE);
   param_validator->add_param("bar", rclcpp::ParameterType::PARAMETER_INTEGER, {0, 10});
 
+  // Create a custom validation callback
+  std::function<rcl_interfaces::msg::SetParametersResult(
```

Comment:
Can you do something to make this line easier to read? It took me at least a minute or two to parse this statement.

Use some using directives to shorten the type names so you can break the line at better points

Reply:
In latest commit, I added the using directives